zone/bonuses.cpp

```diff
@@ -3541,6 +3552,34 @@ void Mob::ApplySpellsBonuses(uint16 spell_id, uint8 casterlevel, StatBonuses *ne
 				new_bonus->Pet_Add_Atk += effect_value;
 				break;
 
+			case SE_ExtendedShielding:
+			{
+				if (AdditiveWornBonus) {
+					new_bonus->ExtendedShielding += effect_value;
+				}
+				else if (effect_value < 0 && new_bonus->ExtendedShielding > effect_value){
+					new_bonus->ExtendedShielding = effect_value;
+        }
```

Comment:
Crazy formatting here

Reply:
merging in git web browser apparently doesn't work well.